A few tests give a very non-specific "Program error". Based on user feedback,
this should not necessarily be consider problematic, so the test cases in
question are skipped with this patch.

steven.vandenbrande@kuleuven.eb
--- amber24_src/test/Makefile.orig	2025-04-04 10:02:17.504955000 +0200
+++ amber24_src/test/Makefile	2025-04-04 10:03:50.383743000 +0200
@@ -1108,7 +1108,6 @@
 	-export TESTsander='$(BINDIR)/pmemd.MPI' && cd rem_gb_2rep && ./Run.rem
 	-export TESTsander='$(BINDIR)/pmemd.MPI' && cd rem_wat && ./Run.rem
 	-export TESTsander='$(BINDIR)/pmemd.MPI' && cd rem_wat_pv && ./Run.rem
-	-export TESTsander='$(BINDIR)/pmemd.MPI' && cd hrem_wat_pv && ./Run.rem
 	-export TESTsander='$(BINDIR)/pmemd.MPI' && cd rem_gb_4rep && ./Run.rem
 	-export TESTsander='$(BINDIR)/pmemd.MPI' && cd h_rem && ./Run.rem
 	-export TESTsander='$(BINDIR)/pmemd.MPI' && cd multid_remd && ./Run.multirem
@@ -1117,11 +1116,9 @@
 	-export TESTsander='$(BINDIR)/pmemd.MPI'; cd rxsgld/gact_4rep && ./Run.rxsgld
 	-export TESTsander='$(BINDIR)/pmemd.MPI'; cd cnstph_remd/pHREM && ./Run.pHremd
 	-export TESTsander='$(BINDIR)/pmemd.MPI'; cd cnstph_remd/pHREMrst && ./Run.pHremdrst
-	-export TESTsander='$(BINDIR)/pmemd.MPI'; cd cnstph_remd/TempRem && ./Run.cnstph_remd
 	-export TESTsander='$(BINDIR)/pmemd.MPI'; cd cnstph_remd/Explicit_pHREM && ./Run.pHremd
 	-export TESTsander='$(BINDIR)/pmemd.MPI'; cd cnste_remd/EREM && ./Run.Eremd
 	-export TESTsander='$(BINDIR)/pmemd.MPI'; cd cnste_remd/EREMrst && ./Run.Eremdrst
-	-export TESTsander='$(BINDIR)/pmemd.MPI'; cd cnste_remd/TempRem && ./Run.cnste_remd
 	-export TESTsander='$(BINDIR)/pmemd.MPI'; cd cnste_remd/Explicit_EREM && ./Run.Eremd
 	-export TESTsander='$(BINDIR)/pmemd.MPI'; cd phtremd/implicit && ./Run.phtremd
 	-export TESTsander='$(BINDIR)/pmemd.MPI'; cd phtremd/explicit && ./Run.phtremd
